backend/data/processors/manual_fixes/CVENFixes.py

```diff
@@ -0,0 +1,163 @@
+    for course in ("CVEN4002", "CVEN4003"):
+        CONDITIONS[course][PROCESSED] = CVEN_4002_4003()
+
+    CONDITIONS["CVEN4050"][PROCESSED] = CVEN_4050()
+    CONDITIONS["CVEN4106"][PROCESSED] = CVEN_4106()
+
+    for course in ("CVEN4201", "CVEN4202", "CVEN4204"):
+        CONDITIONS[course][PROCESSED] = CVEN_4201_4202_4204()
+
+    CONDITIONS["CVEN4300"][PROCESSED] = CVEN_4300()
+    CONDITIONS["CVEN4308"][PROCESSED] = CVEN_4308()
+    CONDITIONS["CVEN4309"][PROCESSED] = CVEN_4309()
+    CONDITIONS["CVEN4507"][PROCESSED] = CVEN_4507()
+    CONDITIONS["CVEN4951"] = CVEN_4951()
+    CONDITIONS["CVEN9826"] = CVEN_9826()
+
+    # Updates the files with the modified dictionaries
+    data_helpers.write_data(
+        CONDITIONS, "data/final_data/conditionsProcessed.json")
+    data_helpers.write_data(COURSES, "data/final_data/coursesProcessed.json")
+
+# TODO: implement your functions here
+def CVEN_2303_3303_3304():
+    """
+    "original": "Prerequisites: (ENGG1300 AND ENGG2400) OR (CVEN1300 AND CVEN2301)<br/>(Pre-requisite only applicable to UG cohort taking this course).<br/><br/>",
+    "processed": "(ENGG1300 && ENGG2400) || (CVEN1300 && CVEN2301) (only applicable to UG cohort taking this course)"
+    """
+
+    return {
+        "original": "Prerequisites: (ENGG1300 AND ENGG2400) OR (CVEN1300 AND CVEN2301)<br/>(Pre-requisite only applicable to UG cohort taking this course).<br/><br/>",
+        "processed": "(ENGG1300 && ENGG2400) || (CVEN1300 && CVEN2301)",
+        "warning": "Requirements only applicable to undergrads"
+    }
+
+def CVEN_4002_4003():
+    """
+    "original": "Pre Req: 132UOC<br/><br/>",
+    "processed": "Req: 132UOC"
+    """
+
+    return "132UOC"
+
+def CVEN_4050():
+    """
+    "original": "Prerequisite: 132 UOCs needed to enrol into this course and BE Hons Programs only<br/><br/>",
+    "processed": "132UOCs needed to enrol into this course && BE Hons Programs only"
+    """
+    # TODO: JOEL: How to say 'BE Hons Programs only", without listing them like this??
+    # Got it from majors here: https://www.handbook.unsw.edu.au/undergraduate/programs/2022/3707?year=2022
+
+    # Additionally it still fails because I haven't said '132UOC in XXXX', what's the command for that???
+    return "132UOC && (AEROAH || BINFAH || CEICAH || CEICDH || COMPBH || CVENAH || CVENBH || ELECAH || ELECCH || GMATDH || MANFBH || MECHAH || MINEAH || MTRNAH || PETRAH || SENGAH || SOLAAH || SOLABH || TELEAH)"
+
+def CVEN_4106():
+    """
+    "original": "Prerequisite: CVEN2101 & CVEN3101<br/><br/>",
+    "processed": "CVEN2101 & CVEN3101"
+    """
+
+    return "CVEN2101 && CVEN3101"
+
+def CVEN_4201_4202_4204():
+    """
+    "original": "Prerequisite/s: CVEN2201 OR CVEN3202 & CVEN3201 OR CVEN3203<br/><br/>",
+    "processed": "CVEN2201 || CVEN3202 & CVEN3201 || CVEN3203"
+    """
+
+    return "(CVEN2201 || CVEN3202) && (CVEN3201 || CVEN3203)"
+
+def CVEN_4300():
+    """
+    "original": "Pre-requisite: 2303, 3303 & 3304<br/><br/>",
```

Comment:
yea, with stuff like this - its everywhere. If you have a good idea of how to automate this fix, you should try your hand at it in conditions_preprocessing.py